spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -2590,15 +2634,17 @@ you would add the following configuration:
 
 
 [[howto-create-a-nonexecutable-jar]]
-=== Create a non-executable JAR with exclusions
-Often if you have an executable and a non-executable jar as build products, the executable
-version will have additional configuration files that are not needed in a library jar.
-E.g. the `application.yml` configuration file might excluded from the non-executable JAR.
+=== Create a Non-executable JAR with Exclusions
+Often, if you have an executable and a non-executable jar as build products, the executable
+version has additional configuration files that are not needed in a library jar.
+For example, the `application.yml` configuration file might by excluded from the non-executable JAR.
 
-The `maven-jar-plugin` used to expose a `forceCreation` attribute that allows you to
+The `maven-jar-plugin` exposed a `forceCreation` attribute that let you
 create the jar _again_ once the `repackage` goal has ran. Arguably, this was a bit fragile
-anyway since it was relying on the order of plugin executions. In Maven, the executable
-jar must be the main artifact and you can add a classified jar for the library:
+anyway, since it was relying on the order of plugin executions. In Maven, the executable
+jar must be the main artifact and you can add a classified jar for the library, as follows:
+// TODO Some of the original content had past tense. Has the `forceCreation` attribute
+// been deprecated? If not, I'll re-arrange the sentences to indicate that it still works.
```

Comment:
I'm not sure why it's written in the past tense. The property still exists and has not been deprecated as far as I can tell: https://maven.apache.org/plugins/maven-jar-plugin/jar-mojo.html#forceCreation.

Reply:
I removed the first part of that paragraph, since it seemed to be disconnected from other content (we mentioned the force-creation attribute and then didn't use it in the example or anywhere else. If we need to restore, we need to think of something useful to say about it.

Reply:
LGTM. Thanks.